### ISSUE TITLE
[Enhancement]: Should have delete option for the uploaded picture/voice wave 

### ISSUE THREAD

**Issue by rleitan** [mozilla/voice-web#2176]
[STR]:
1.Upload an image as avatar to your profile
2.Record and upload a voice wave 

[Expected result]:
Should have delete option for the uploaded picture/voice wave

@mbransn what do you think?

**Comment by mbransn** [comment]
@rleitan yes, agreed 100% -- I've noted this on zeplin screens where they should be and cc'd you + @viorelaioia and @Gregoor

**Comment by moz-bozden** [comment]
Closing as stale...
